polling_stations/templates/base.html

```diff
@@ -46,8 +46,8 @@
         {% endblock site_meta %} 
 
     </head>
-
-    <body {% block body_attrs %}{% endblock body_attrs %} class="ds-scope">
+    {% block body_base %}
+    <body {% block body_class_attrs %}{% endblock body_class_attrs %} class="ds-scope">
```

Comment:
This can remain named as `{% block body_attrs %}` because you might want to add other attrs rather than just the class. But more importantly, your default `class="ds-scope"` is defined outside of the block. So this means when you do use the block in the `base_embed.html` template, you are not overriding the existing attrs, you are addinging additional ones (So you would have `<body class="embed" class="ds-scope"`.

So change to:
```
<body {% block body_class_attrs %}class="ds-scope"{% endblock body_class_attrs %}>

Reply:
Yes thanks, caught this one after I pushed.